rpcs3/Emu/Cell/Modules/cellVdec.cpp

```diff
@@ -524,6 +543,11 @@ error_code cellVdecClose(ppu_thread& ppu, u32 handle)
 		return CELL_VDEC_ERROR_ARG;
 	}
 
+	if (vdec->current_state == CELL_VDEC_MSG_TYPE_AUDONE)
```

Comment:
Maybe make it thread safe by adding a reader/shared lock for each check and writer lock when writing to current_state?

Reply:
nah. I don't wanna test the game again XD

Reply:
I removed the current_state stuff as it was causing regressions.
And it appears that Lair was just mocking me by randomly working every time i booted with the changes.